starter-web-netty/build.gradle

```diff
@@ -1,47 +1,30 @@
 plugins {
     id "io.micronaut.internal.starter.convention"
     id "com.github.johnrengelman.shadow"
-    id "application"
+    id 'io.micronaut.application' version '3.4.0'
```

Comment:
It looks like the plugin is already included in the settings so you don't need to put the version here